### PR TITLE
adds parallel finance tvl

### PR DESCRIPTION
**Pull request by drbh**
##### Twitter Link:
https://twitter.com/ParallelFi

##### Website Link:
https://parallel.fi/

##### Current TVL:
$766,565,684.39

##### Short Description (to be shown on DefiLlama):
Parallel Finance is a Decentralized Money Market Protocol that offers lending, staking, and borrowing in the Polkadot ecosystem. Depositors can lend and stake simultaneously to earn double yield on their staked coins, and borrowers can collateralize to borrow.

### CODE REVIEW

**Comment by drbh** [comment]
@0xngmi please let me know if any changes should be made

**Comment by 0xngmi** [comment]
is this just using crowdloan locked tokens as tvl? if so we can't use it, we are not counting crowdloans as tvl

**Comment by drbh** [comment]
@0xngmi The TVL does not include the value of the tokens contributed to the Parallel crowdloan. 

The TVL shown is the current value of all Polkadot that has been locked in to Parallel accounts by users. This DOT is later used to contribute to active Polkadot crowdloans on behalf of the user request but is held and managed by Parallel.

TLDR; This TVL is the total value of DOT transferred to Parallel accounts.

**Comment by drbh** [comment]
@0xngmi bump, and Happy New Year! 🎊 🍾

**Comment by waynebruce0x** [comment]
Does this count only funds that are waiting to be deployed to crowdloans, or does it also include funds deposited into crowdloans on behalf of the users?

**Comment by drbh** [comment]
@waynebruce0x thanks for the response. They are the capital that is locked in Parallel system and contribute on behalf of the users. They are users funds that use Parallel to contribute to other crowdloan projects. Parallel the contract level that manage their funds. 

More than 50-100k users put their DOT to Parallel and chose to lock up for 96 weeks in order to get project token rewards. 

My understandings is that typical crowdloan doesn’t count since the DOT is locked in Relaychain. In this case, DOT is locked in Parallel.

Please let me know if hopping on a call would be helpful! Thanks

**Comment by drbh** [comment]
Hey guys, just following up on this PR. Would it be possible to schedule a quick chat with our team to discuss how we secure our TVL?

**Comment by yubo-ruan** [comment]
@0xngmi @waynebruce0x Hi it was great to meet you and thank you so much for working with us. I'm the founder of Parallel finance and I'd like to add on some input here. Users locked up their funds in Parallel protocol first and then we manage their DOT to lock up again in Polkadot Relaychain. To date, over 25M DOT (2.5% of DOT total supply) are locked in the contract of a parallel crowdloan pallet. We also issue cDOT (which is very similar to stETH or cvxCRV) to all users in order to provide liquidity on our CDP and AMM protocol. There is more information to share but could we arrange a call and I will explain to you in detail? 

Parallel finance is determined to bring DeFi to one billion people and we are backed by Sequoia, Peter Thiel's Founders Fund, and Polychain. I hope we can work with defillama in the long term as we are going to deploy 5 more protocols on Polkadot space and potentially a few more protocols on Ethereum L2.

**Comment by waynebruce0x** [comment]
Hey thanks for explaining and sorry for the delayed response. We don't count TVL for crowdloans, but since you offer a liquid staking product sitting on top of the crowdloans I think we should include you in our dashboard.

**Comment by waynebruce0x** [comment]
I understand this is counting the value of all the DOT deposited to Parallel? is that correct?

**Comment by yubo-ruan** [comment]
This is correct!

**Comment by yubo-ruan** [comment]
Thank you so much! Let me know if you have questions

**Comment by yubo-ruan** [comment]
@waynebruce0x Are we counted as a DeFi App or a chain? FYI, we are an L1 chain powered by the Substrate framework.

**Comment by yubo-ruan** [comment]
Hi @waynebruce0x thank you so much for listing us and I saw the latest [page](https://defillama.com/protocol/parallel-finance). However, Parallel is an L1 chain that we have our own validators/collators (similar to Moonriver and Karurua) instead of a protocol. Could you move us to the chains section? Really appreciate it!

**Comment by waynebruce0x** [comment]
Sure we will change it so you're your own chain

**Comment by yubo-ruan** [comment]
Sorry to bother you again but it seems like it is still [showing](https://defillama.com/chain/Polkadot) us as Polkadot, not Parallel and Stafi (FIS) is NOT part of our L1. Also, could you change it to Parallel Liquid Crowdloan Protocol with our logo instead of Liquid staking with smiley face emoji?

**Comment by yubo-ruan** [comment]
Thank you for changing for us! Really appreciate that. Just a quick thing - could you change the name to Parallel Liquid Crowdloan?  

image
